src/main/scala/dpla/ingestion3/OaiRecordHarvesterMain.scala

```diff
@@ -121,6 +118,11 @@ object OaiRecordHarvesterMain {
       case (Some(sets), Some(blacklist)) => {
         baseOptions + ("sets" -> (sets.split(",") filter (!blacklist.split(",").contains(_))).mkString(","))
       }
+      // No set list passed in but a blacklist was passed in
+      // TODO: Should this error out or should it try to call ListSets to generate a set list?
```

Comment:
I guess the assumption here is that the notion of "sets" should have been resolved before invoking the OaiHarvestMain. To me, it feels like this might be informed by how we set up the refactor of Audrey's work (splitting this into three mains) to include the generation of the sets before invoking the record harvest.  Options 1 - we plan for ListSets to invoked before harvesting records which would simplify the record harvest because sets are passed in as comma separated string if they are provided as a manual "whitelist" in the profile or command line or by calls to ListSets. 

Or we only make a call to ListSets if they gave us a blacklist and no sets.  IMO the first seems option seems more straightforward. 

Not sure how well I've explained my thinking here..